src/Kestrel.Transport.Abstractions/Internal/TransportConnection.FeatureCollection.cs

```diff
@@ -20,6 +20,8 @@ public partial class TransportConnection : IHttpConnectionFeature,
                                                IApplicationTransportFeature,
                                                ITransportSchedulerFeature,
                                                IConnectionLifetimeFeature,
+                                               IConnectionHeartbeatFeature,
+                                               IConnectionLifetimeNotificationFeature,
```

Comment:
Nit: implement the methods for these interfaces explicitly in this file for consistency. It can be passthrough.